.github/CODEOWNERS

```diff
@@ -489,7 +489,8 @@ Makefile                                              @thiagokokada
 
 /modules/services/parcellite.nix                      @gleber
 
-/modules/services/pass-secret-service.nix             @cab404
+/modules/services/pass-secret-service.nix             @cab404 @cyntheticfox
+/tests/modules/services/pass-secret-service.nix       @cyntheticfox
```

Comment:
What file is this?

``` console
$ ls tests/modules/services/pass-secret-service.nix
No such file or directory
```

Reply:
Maybe you meant

    /tests/modules/services/pass-secret-service       @cyntheticfox